index.html

```diff
@@ -29,6 +33,21 @@ <h1>Happy Hearts Roaming</h1>
               <a href="#asia">Asia</a>
             </div>
           </div>
+          <div class="search-container">
```

Comment:
The input color should be white. Also a bit more padding

Reply:
Yes, I adjusted the color and the padding